test/methods.js

```diff
@@ -400,6 +400,32 @@ asyncTest("remote", function() {
 	strictEqual( v.element(e), true, "still invalid, because remote validation must block until it returns; dependency-mismatch considered as valid though" );
 });
 
+asyncTest("remote, pending class added to element while call outstanding", function() {
```

Comment:
I forgot to mention the spacing issue in this test as well.
Please fix them as I mentioned above